### DISCUSSION TITLE
R13 Minimum TABLE requirements

### DISCUSSION THREAD

**Discussion by dcooley** [mozman/ezdxf#973] [Q&A]
The documentation for R13 tables (https://ezdxf.readthedocs.io/en/stable/dxfinternals/filestructure.html#dxf-r13-r14-and-later) states

> VPORT entry *ACTIVE is not required! Empty table is ok for AutoCAD.

Does this mean that the `VPORT` table can be empty, or the `*ACTIVE` table can be empty?

In the `MINIMAL_DXF_AC1021` file there is a `*ACTIVE` entry - https://github.com/mozman/ezdxf/blob/master/examples_dxf/Minimal_DXF_AC1021.dxf#L50

If this is the 'minimal' requirements for the AC1021 version, should the `*ACTIVE` entry be empty / not there at all?

**Comment by mozman** [comment]
Just try it. DXF is a text format you can edit the file with any text editor. Than open the file with `AutoCAD` or `DWG Trueview` and look what happens.

**Comment by dcooley** [comment]
I tried opening it in AutoCad's online viewer but it fails.
